programs/locking/src/lib.rs

```diff
@@ -100,70 +99,33 @@ mod locking {
                 },
                 signer,
             ),
-            mint_amount,
+            amount,
         )?;
 
-        emit!(EventStake {
-            mint_amount,
+        emit!(EventLock {
+            mint_amount: amount,
             token_amount: amount,
```

Comment:
They are the same, so we just need 1 field

Reply:
Fixed